include/notification.h

```diff
@@ -57,7 +58,8 @@ enum mako_notification_close_reason {
 	MAKO_NOTIFICATION_CLOSE_EXPIRED = 1,
 	MAKO_NOTIFICATION_CLOSE_DISMISSED = 2,
 	MAKO_NOTIFICATION_CLOSE_REQUEST = 3,
-	MAKO_NOTIFICATION_CLOSE_UNKNOWN = 4,
+	MAKO_NOTIFICATION_CLOSE_REPLACED = 4,
+	MAKO_NOTIFICATION_CLOSE_UNKNOWN = 5,
```

Comment:
We can't do this, because the reason is part of the spec.

"Replacing" a notification should behave (from the user PoV) as if the original notification had been updated, not closed.